### DISCUSSION TITLE
阅读源码的意义与方法 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#238] [Announcements]
https://mazhuang.org/2017/10/05/rtfsc-methods-and-meaning/ 

 思索了这两个问题良久，也去知乎找了一些相关话题的问答，但并没有标准答案。所以，我这里也只是记录一些我对此的看法，也许会随着 RTFSC 阅历的丰富而发生变化，我会记录更新于 https://github.com/mzlogin/rtfsc-android。

**Comment by zdx0823** [comment]
哈哈哈

**Comment by zdx0823** [comment]
## 哈哈哈
